opensquirrel/passes/decomposer/swap2cnot_decomposer.py

```diff
@@ -2,7 +2,7 @@
 
 from typing import TYPE_CHECKING
 
-from opensquirrel.ir import CNOT
+from opensquirrel.ir.default_gates import CNOT
```

Comment:
Import gates directly from `opensquirrel` (where possible).

```suggestion
from opensquirrel import CNOT
```

Reply:
Done!